src/main/java/seedu/address/model/person/Person.java

```diff
@@ -14,6 +14,7 @@ public class Person {
     // Identity fields
     private final Name name;
     private final Phone phone;
+    private final NokPhone nokPhone;
```

Comment:
I have previously abstracted `Money` out of the `MoneyOwed` and `MoneyPaid` as both fields are essentially referring to the same type of data and there should not be a need to separate them from each other. I think similar issue applies for here? Since both`NokPhone` and `Phone` are just `Phone` class and when we have a different requirement for the `Phone` class, for eg supporting '-' or '+' sign in the value, both sides should be updated accordingly. Hence, I think it makes more sense and becomes more OOP if we have only one `Phone` data type and use it to represent both `NokPhone` and `Phone`.

Reply:
This abstraction is also scheduled for upcoming PR and hence is resolved for now.